packages/pdf-renderer/README.md

```diff
@@ -1,7 +1,79 @@
 # pdf-renderer
 
-This library was generated with [Nx](https://nx.dev).
+pdf-renderer is a NestJS module used for creating pdf files out of react components.
 
-## Running unit tests
+## Installation
 
-Run `nx test pdf-renderer` to execute the unit tests via [Jest](https://jestjs.io).
+```
+npm install @leancodepl/pdf-renderer
+yarn add @leancodepl/pdf-renderer
+```
+
+## Prerequisites
+
+For the pdf-renderer to run correctly, it is required to use Dockerfile delivered with a package, which installs all of
+the dependencies needed by Puppeteer. <br/>
+
+If you want to provide custom fonts, you need those in the `.woff` format, which is the only one supported at the
+moment. <br/>
+
+Keep in mind, that everything you pass into the renderer, has to be synchronous, for it to work as expected. Any data
+component needs, should be provided beforehand. Also, component shouldn't call any hooks.
+
+## Usage
+
+To use pdf-renderer you have to register it inside a module of your own NestJS app. Register takes a single argument, an
+object with a `fontsConfiguration` field. If needed, `FontsConfiguration` type can be imported from the package.
+
+```
+export const OpenSansRegular = Symbol("OpenSansRegular");
+
+const fontsPath = path.join(__dirname, "path_to_fonts_folder");
+
+const fontsConfiguration: FontsConfiguration = {
```

Comment:
You could also add from where to import this and `PdfRenderer` because it's not shown in this example.